onward/app/models/OnwardCollection.scala

```diff
@@ -64,6 +65,7 @@ object OnwardCollection {
         designType = content.properties.maybeContent.map(_.metadata.designType).getOrElse(Article).toString,
         webPublicationDate = content.webPublicationDate.withZone(DateTimeZone.UTC).toString,
         headline = content.header.headline,
+        mediaType = content.card.mediaType.map { m => m.toString() }
```

Comment:
What you've written here is probably clearer but there's a lazier way of writing this: `mediaType.map(_.toString())

Reply:
Ah thanks, good spot :+1: